### PR TITLE
Samtools sort

### PR DESCRIPTION
**Pull request by emmarousseau**
## Description

Samtools sort

## Checklist before requesting a review

- [x] I have performed a self-review of my code

- [x] Conforms to the [Contributing guidelines](https://github.com/viash-hub/base/blob/main/CONTRIBUTING.md)

- [x] Proposed changes are described in the [CHANGELOG.md](https://github.com/viash-hub/base/blob/main/CHANGELOG.md)

- [x] I have tested my code with `viash ns test --parallel -q <name or namespace>`

- Check the correct box. Does this PR contain:
  - [ ] Breaking changes
  - [x] New functionality
  - [ ] Major changes
  - [ ] Minor changes
  - [ ] Documentation
  - [ ] Bug fixes

### CODE REVIEW

File: src/samtools/samtools_sort/config.vsh.yaml [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+        required: true
+        must_exist: true
+  - name: Outputs
+    arguments:
+      - name: --output
+        type: file
+        description: |
+          Write final output to file.
+        required: true
+        must_exist: false
```
Comment (rcannood) [inline src/samtools/samtools_sort/config.vsh.yaml:28]: Output should always be written to file, not to stdout

```suggestion
```

**Review by rcannood — CHANGES_REQUESTED** [review]
minor suggestions ^^

File: src/samtools/samtools_sort/config.vsh.yaml (hunk 1 above)
Comment (emmarousseau) [reply src/samtools/samtools_sort/config.vsh.yaml]: Hey, isn't this how we make sure it's written to a file but without requiring it to exist before running to command? Without this line we need to have an empty file beforehand to which we write the output. This is how I did it in previous components. Please let me know! thanks

Comment (rcannood) [reply src/samtools/samtools_sort/config.vsh.yaml:28]: Correction: the direction should be set to `output`

```suggestion
        direction: output
```

File: src/samtools/samtools_sort/test.sh [hunk 2]
```diff
@@ -0,0 +1,79 @@
+#!/bin/bash
+
+test_dir="${meta_resources_dir}test_data"
+out_dir="${meta_resources_dir}test_data/text"
```
Comment (rcannood) [inline src/samtools/samtools_sort/test.sh:4]:
```suggestion
test_dir="${meta_resources_dir}/test_data"
out_dir="${meta_resources_dir}/test_data/text"

**Review by rcannood — APPROVED** [review]

**Review by rcannood — APPROVED** [review]

**Pull request merged**